Create migration page

mJarsater opened the pull request:
**Describe pull-request**  
Created a page for the migration where we can log breaking changes.

**Solving issue**  
Fixes: AB#2494

**How to test**  
1. Go to storybook link below.
2. Check in Intro -> Migration

**Screenshots**  
<img width="1509" alt="Screenshot 2022-11-04 at 12 57 43" src="https://user-images.githubusercontent.com/62651103/199967244-4a985442-3f11-433f-a754-52bb8ff35a98.png">


**Additional context**  
-

theJohnnyMe reviewed (approved):
Personally, I might prefer using markdown over HTML as it is a bit easier to read while you are editing, but this can be the first version. 
The only thing that might be useful is to have [this one ](https://www.markdownguide.org/cheat-sheet/) as a comment somewhere so everyone can pick it up.

mJarsater commented:
> Personally, I might prefer using markdown over HTML as it is a bit easier to read while you are editing, but this can be the first version. The only thing that might be useful is to have [this one ](https://www.markdownguide.org/cheat-sheet/) as a comment somewhere so everyone can pick it up.

I agree. I have updated the PR and added the link you posted in the migration.md aswell.
